apps/web/components/pico-logo.tsx

```
@@ -0,0 +1,16 @@
+import { Bot } from "lucide-react";
+import React from "react";
+import Link from "next/link";
+
+export const PicoLogo = ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)